app/javascript/utils/outOfFocus.js

```diff
@@ -0,0 +1,24 @@
+$(() => {
+  // This bit of code currently only closes dropdown menus in a table when the focus is no longer in the containing cell.
```

Comment:
Thanks for adding all of this commentary @johnpinto1 It will be helpful in the future!